px-kpi-list.html

```diff
@@ -50,26 +52,26 @@
 <dom-module id="px-kpi-list">
   <template>
     <style include="px-kpi-styles"></style>
-
-
-    <div class="flex flex--row flex--middle u-mv--">
-      <div class="flex flex--left flex__item">
-        <span class="caps epsilon secondary u-mv0">{{label}}</span>
+    <template is="dom-if" if="{{_checkIsHeaderDataPresent()}}">
```

Comment:
This doesn't work. It checks if `label`, `statusIcon` or `statusLabel` is present _at the time it is rendered_, then never checks again.

This breaks the predix-ui demo:
<img width="500" alt="px-kpi regression" src="https://user-images.githubusercontent.com/4055224/46992936-58af2280-d158-11e8-8c0e-9461c3962b60.png">

Refactoring this to a computed property should resolve this: https://www.polymer-project.org/2.0/docs/devguide/observers#define-a-computed-property